lib/basedriver/helpers.js

```diff
@@ -27,19 +27,14 @@ async function configureApp (app, appExt, mountRoot="Volumes", windowsShareUserN
 
   if ((app || '').substring(0, 4).toLowerCase() === 'http') {
     logger.info(`Using downloadable app '${app}'`);
-    let downloadedApp = await downloadApp(app, shouldUnzipApp ? '.zip' : appExt);
+    let downloadedApp = await downloadApp(app, appExt);
     newApp = downloadedApp.targetPath;
```

Comment:
Would it? There would be a name mismatch (since the semantics of the returned path and what the calling code uses it for are distinct) so there would still be an assignment outside the deconstruction step. Since it's a mix of local variables and ones from the enclosing scope it would also need another step:
```js
let targetPath;
({targetPath, shouldUnzipApp} = await downloadApp(app, appExt));
newApp = targetPath;
```

I do think, semantically, the `shouldUnzipApp` of the returned object needs to be renamed, so part of that would be gone. Something like:
```js
let {targetPath, isZipFile} = await downloadApp(app, appExt);
newApp = targetPath;
shouldUnzipApp = isZipFile;
```